package.json

```diff
@@ -27,7 +27,7 @@
   },
   "devDependencies": {
     "@okp4/eslint-config": "^1.1.0",
-    "@okp4/ui": "^2.0.0",
+    "@okp4/ui": "^3.0.0",
```

Comment:
This is not in the scope of the PR. We should wait until #74 is merged before rebasing this PR and then we can review